sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.cs

```diff
@@ -27,7 +30,35 @@ public void Test()
             string myPassword1 = null;
             string myPassword2 = "";
             string myPassword3 = "        ";
-            string myPassword4 = @"foo"; // Noncompliant {{'password' detected in this expression, review this potentially hardcoded credential.}}
+            string myPassword4 = @"foo"; // Noncompliant
+        }
+
```

Comment:
I suggest to add
private const string Secret= "LoremIpsum";
and
new NetworkCredential("username", Secret); // Noncompliant
to demonstrate that .IsConstant(..) is working.

Reply:
Example added in unit test